dbtc/client/cloud/base.py

```diff
@@ -77,10 +59,28 @@ def _header_property(self):
 
         return 'api_key'
 
+    def _clone_resource(self, resource: str, **kwargs):
+        create_args = kwargs.pop('create_args', None)
+        payload = getattr(self, f'get_{resource}')(**kwargs)['data']
+
+        # Can't recreate a resource with an ID
+        payload.pop('id', None)
+        if create_args is not None:
+            kwargs = {k: v for k, v in kwargs.items() if k in create_args}
+        kwargs['payload'] = payload
+        return getattr(self, f'create_{resource}')(**kwargs)
+
     def _make_request(
         self, path: str, *, method: str = 'get', **kwargs
     ) -> requests.Response:
         """Make request to API."""
+
+        # Model is not an argument that the request method accepts, needs to be removed
+        model = kwargs.pop('model', None)
```

Comment:
@matt-winkler This is where we would pull out the `model`, if it's been passed.  And line 83 below will assign the validated model, with the appropriate defaults, back to the `json` key.